integration/mediation-tests/tests-transport/pom.xml

```diff
@@ -335,6 +335,12 @@
             <groupId>io.netty</groupId>
             <artifactId>netty-resolver</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.icegreen</groupId>
+            <artifactId>greenmail</artifactId>
+            <version>1.3</version>
```

Comment:
Can we use a property without hardcoding the version here?. Or we can set the dependency version from main pom and use it here.

Reply:
done